maintenance/wikia/PortabilityMetric/updateMainPageFilterInMetricProcessor.php

```diff
@@ -0,0 +1,41 @@
+<?php
+
+/**
+ * @ingroup Maintenance
+ *
+ * @desc For current wiki, find an ID of it's Main Page
```

Comment:
I think that the last part missing is actually adding the execution this script to cron, isn't it?